lib/order.rb

```diff
@@ -0,0 +1,135 @@
+require 'csv'
+require 'awesome_print'
+require_relative 'customer.rb'
+
+class Order
+
+attr_reader :id
+
+# The fulfillment_status = :pending sets a default value for fulfillment_status
+# Note: @products is a single hash that contains key: food_name and value: price
+# Example: @products = {"banana" => 2.25, "cookie" => 1.50}
+# The constructor's customer parameter is a Customer object NOT customer id
+def initialize(id, products, customer, fulfillment_status = :pending)
+  @id = id
+  @products = products
+  @customer = customer
+
+  if fulfillment_status == :pending || fulfillment_status == :paid || fulfillment_status == :processing || fulfillment_status == :shipped || fulfillment_status == :complete
+    @fulfillment_status = fulfillment_status
+  else
+    raise ArgumentError, 'An invalid fulfillment status value has been entered'
+  end
+end
+
+def id
+  return @id
+end
+
+def products
+  return @products
+end
+
+def customer
+  return @customer
+end
+
+def fulfillment_status
+  return @fulfillment_status
+end
+
+def total
+  total_wo_tax = 0
+  tax_rate = 0.075
+
+  # Iterating through products hash and summing all the values
+  @products.each do |k,v|
```

Comment:
I tend to prefer variable names besides `k` and `v`! What do the keys and value pairs in this hash represent?